### DISCUSSION TITLE
How to reopen an issue?

### DISCUSSION THREAD

**Discussion by MatrixManAtYrService** [nushell/nushell#11617] [General]
I've been adding a sort of wish-list to this issue: https://github.com/nushell/nushell/issues/2878#issuecomment-1826211941

I hope it's helpful and not seen a list of demands--I very much appreciate the work you guys do here. I reference this repo in conversations as an example of how active and vibrant the Rust community is.  I've been learning Nim but you guys are tempting me to switch for Rust.

It seemed like the kind of thing that's hard to think of when you run across the issue, but easy to think of if you're using vi-mode extensively and run across an omission.

Trouble is, the issue is closed.  Github actions says:

> This issue has been marked stale for more than 10 days without activity. Closing this issue, but if you find that the issue is still valid, please reopen.

I don't have a button to reopen.  Does somebody else have that button?  Would it be better to make a new issue?

Also, I've been wanting to learn Rust.  I might need my hand held a little bit, but I'm willing to take a crack at this if somebody thinks its a good first issue and can point me in the right direction.

**Comment by fdncred** [comment]
What you should probably do is create a new issue with tracking checkboxes of the things you're asking for. Either that or create a separate issue for each item. Having 100 things in 1 issue makes it difficult to accomplish and hard to consume.
